packages/launchpad/src/setup-actions.ts

```diff
@@ -21,7 +21,7 @@ export const setUpActionsForTasks = ( {
 	uiContext = 'calypso',
 }: LaunchpadTaskActionsProps ): Task[] => {
 	const { recordTracksEvent, checklistSlug, tasklistCompleted, launchpadContext } = tracksData;
-	const { setShareSiteModalIsOpen, siteLaunched } = extraActions || {};
+	const { setShareSiteModalIsOpen, siteLaunched, setActiveChecklist } = extraActions || {};
```

Comment:
I think it would be better if `setActiveChecklist` was wired in more directly, but given that we're not in a hook or component, we likely need it passed in. That said, I think we might want to push that to a different argument that's required rather than optional, as I don't see any use cases where we would _not_ want the active checklist set for the site.